runtimes/adapters/prime_agent.py

```
@@ -0,0 +1,519 @@
+            proc = await asyncio.create_subprocess_exec(
+                bin_path, "--help",
+                stdout=asyncio.subprocess.PIPE,
+                stderr=asyncio.subprocess.PIPE,
+                stdin=asyncio.subprocess.DEVNULL,
+            )
+            stdout, _ = await asyncio.wait_for(proc.communicate(), timeout=15.0)
+            for token in stdout.decode(errors="replace").split():
+                token = token.strip(",")
+                if token.startswith("--"):
+                    flags.add(token)
+        except Exception as exc:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)